dysnix/bsc/templates/statefulset.yaml

```diff
@@ -237,6 +237,25 @@ spec:
         resources:
           {{- toYaml .Values.metrics.resources | nindent 10 }}
       {{- end }}
+      {{- if .Values.metricsExtra.enabled }}
+      - name: metricsadditional
```

Comment:
Could we please use some meaningful container name here?

Reply:
Rename to timestamp-monitor but maybe you have other suggestion  ?